openwisp-config/files/openwisp.agent

```diff
@@ -95,6 +96,10 @@ FETCH_COMMAND="curl -s --connect-timeout $CONNECT_TIMEOUT --max-time $MAX_TIME"
 mkdir -p $WORKING_DIR
 mkdir -p $UNMANAGED_DIR
 
+if [ -f "$PERSISTENT_CHECKSUM" ]; then
+	cp "$PERSISTENT_CHECKSUM" "$CONFIGURATION_CHECKSUM"
```

Comment:
what happens the first time this is run and there's no persistent checksum? Does it generate an error/warning in logread? If yes can you handle it so it doesn't pelase?

Reply:
The if statement ensures that a persistent checksum is present.